aleph2_analytic_services_storm/src/main/java/com/ikanow/aleph2/analytics/storm/utils/StormControllerUtil.java

```diff
@@ -232,7 +232,7 @@ public static IStormController getStormControllerFromYarnConfig(String yarn_conf
 						.filter(f -> !f.equals(""))
 						.collect(Collectors.toSet()));
 				
-				if (jars_to_merge.isEmpty()) { // special case: no aleph2 libs found, this is almost certainly because this is being run from eclipse...
+				if ( isOnlyHadoopDep(jars_to_merge)) { // special case: no aleph2 libs found, this is almost certainly because this is being run from eclipse...
```

Comment:
lol gross, a more generic way of doing this would probably to be to check that 1+ startsWith("aleph2_") JAR exist (or just aleph2_data_model). That way if some other random JAR finds its way in then it won't break everything (especially since i don't think our test coverage for storm is great at the moment)